cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,23 @@
 
 describe('Student Registration page', () => {
   before(() => {
-    
+    cy.visit('/');
   });
 
-  it('', () => {
+  it('should register after inputting valid information', () => {
+    cy.get('#firstName').click().type('Eka');
+    cy.get('#lastName').click().type('One');
+    cy.get('#userEmail').click().type('test@i.ua');
+    cy.get('#gender-radio-2').click({force: true});
+    cy.get('#userNumber').click().type('0637959971');
+    cy.get('#dateOfBirthInput').click().type('{selectAll}').type('11 Nov 2022').type('{enter}');
+    cy.get('.subjects-auto-complete__value-container').click().type('Math'+ '{enter}');
+    cy.get('[for="hobbies-checkbox-2"]').click();
+    cy.get('#currentAddress').click().type('Kyiv');
+    cy.get('#state').type('raj'+ '{enter}');
+    cy.get('#city').type('jai' + '{enter}');
+    cy.contains('#submit', 'Submit').click({force:true});
+    cy.contains('[class="modal-title h4"]', 'Thanks for submitting the form');
```

Comment:
you also need to assert the data in the modal window
<img width="795" alt="image" src="https://user-images.githubusercontent.com/35568237/208428331-75fbe312-b1ea-402a-bca3-f4c7249dd5ed.png">

Reply:
fixed)

Reply:
or I'm blind, or not fixed 🫠